robot/rospackages/src/image_scp/config.txt

```diff
@@ -0,0 +1,6 @@
+#Use this as a template for entering user details and destination of files so script doesn't need to be changed.
+#Enter credentials/ path without quotation marks 
+Username replaceWithUse
```

Comment:
you meant replaceWithUser? I would suggest to make it more clear like `basestationUserName`, though perhaps (since I recommend using a python file) it can be set automatically, by getting the `$USER` environement variable through python.

Reply:
I added instructions for testing,  do they make sense?